core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDLPAction.java

```diff
@@ -61,7 +59,7 @@ public boolean execute(Context context) throws Exception {
       case Event.PROPERTY_ADDED:
       case Event.PROPERTY_CHANGED:
         PropertyImpl property = (PropertyImpl) context.get(InvocationContext.CURRENT_ITEM);
-        if(property != null && !property.getName().equals(EXO_EDITORS_RUNTIME_ID) && !property.getName().equals(FileDlpConnector.RESTORE_WORKSPACE) && !property.getName().equals(FileDlpConnector.RESTORE_PATH)) {
+        if (property != null && !property.getName().equals(EXO_EDITORS_RUNTIME_ID) && !property.getName().equals(FileDlpConnector.EXO_CURRENT_PROVIDER) && !property.getName().equals(FileDlpConnector.RESTORE_WORKSPACE) && !property.getName().equals(FileDlpConnector.RESTORE_PATH)) {
```

Comment:
This modification should not be necessary

Reply:
It is necessary, because if we delete that, the file will be quarantined once we open and close the editor without making any changes to the file.

Reply:
Ok. Can you rework this part and create a list of excluded property names ? 
So that the if will be 
if (property!=null && !excludedPropertyNames.contains(property.getName()) {
...